### ISSUE TITLE
[med] : Many pages: Chrome : Combo box label is not announced by the screen reader.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8553]
## Action Performed:
1. Using Chrome, open URL staging.new.expensify.com
2. Navigate to Setting using Tab key and press Enter to select it.
3. Navigate to preferences using Tab key and press using Enter to select it.
4. Navigate to combo boxes using Ta key.
5. Observe the screen reader announcement.

## Expected Result:
Combo box label should be announced by the screen reader. It should read as "Priority mode Focus combo box collapsed"

## Actual Result:
Combo box label is not announced by the screen reader. Screen reader is reading "focus combo box".

**Note**
Same issue repro on all pages where combo boxes are presented.

**Other occurrences**
>Same issue repro on PR #8765

## Workaround:
Yes

## Area issue was found in:
Combo box

## Failed WCAG checkpoints
1.3.1

## User impact:
Screen reader user does not know the purpose of the combo box, if the label is not announced by the screen reader.

## Suggested resolution:
Make sure screen reader announces the label of the combo box. (i.e. Priority mode)
Refer to:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

## Platform:
- Web
- Desktop App

**Version Number:**v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/162358544-0cd417e5-ca46-444c-9f89-2ce4a9527075.mp4
